cinema/migrations/0001_initial.py

```diff
@@ -77,6 +78,12 @@ class Migration(migrations.Migration):
                 ("title", models.CharField(max_length=255)),
                 ("description", models.TextField()),
                 ("duration", models.IntegerField()),
+                (
+                    "image",
+                    models.ImageField(
+                        null=True, upload_to=cinema.models.movie_image_path
+                    ),
+                ),
```

Comment:
The `ImageField` has been correctly added to the `Movie` model. Ensure that the `movie_image_path` function is defined in the `cinema.models` module to specify the upload path for images.